spec/integration/strong_resource_spec.rb

```diff
@@ -132,21 +132,53 @@
           params[:person][:company_attributes] = {
             title: 'walmart',
             state_attributes: {
-              acronym: 'ny'
-            }
+            acronym: 'ny'
+          }
           }
         end
 
         it 'allows nesting relations' do
           expect(controller.strong_resource.to_h)
-            .to eq('name' => 'John', 'company_attributes' => {
-              'title' => 'walmart',
-              'state_attributes' => {
-                'acronym' => 'ny'
-              }
-            })
+          .to eq('name' => 'John', 'company_attributes' => {
+            'title' => 'walmart',
+            'state_attributes' => {
+            'acronym' => 'ny'
+          }
+          })
         end
       end
     end
   end
 end
+
+describe "strong_resources" do
+  context "with multiple definitions" do
+    let(:controller) do
+      controller = CompaniesController.new
```

Comment:
Prefer to define the controllers with `Class.new` within the tests so evals don't accidentally affect other tests